couchbase/README.md

```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
 
 ### USAGE
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
 
 For general reference on how to monitor Couchbase, see <a target="_blank" href="http://blog.couchbase.com/monitoring-couchbase-cluster">Couchbase Monitoring</a> and <a target="_blank" href="http://developer.couchbase.com/documentation/server/4.0/monitoring/monitoring-rest.html">Monitor using the REST API</a>.
 
 **Note on bucket metrics**
 
-This plugin emits some metrics about the bucket's performance across the cluster, and some metrics about the bucket's performance per node.
+This plugin emits metrics about bucket performance across the cluster, and also metrics about bucket performance per node.
 
-Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket. In order to analyze bucket performance for the entire bucket, apply functions like Sum or Mean to group node-level metrics together by bucket.
+Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket.
```

Comment:
is the plug-in doing the reporting here?

Reply:
The nodes are doing the reporting.